std/assembly/number.ts

```diff
@@ -295,7 +295,7 @@ export abstract class F32 {
   }
 
   static parseInt(value: string, radix: i32 = 0): f32 {
-    return <f32>strtol<i64>(value, radix);
+    return <f32>strtol<f64>(value, radix);
```

Comment:
Does `strtol<f32>` suffer from precision issues here?

Reply:
yes

Reply:
also it reduce monomorphic variants to just `strtol<f64>` instead `strtol<f32>` and `strtol<f64>`